test/integration/bundle/bundle.test.ts

```diff
@@ -4,10 +4,11 @@ import path from 'path';
 import { fileCleanup } from '../../helpers';
 
 const spec = fs.readFileSync('./test/integration/bundle/final-asyncapi.yaml', {encoding: 'utf-8'});
+const specv3 = fs.readFileSync('./test/integration/bundle/final-asyncapiv3.yaml', {encoding: 'utf-8'});
 
-function validateGeneratedSpec(filePath: string, spec: string) {
+function assertSpecsEqual(filePath: string, spec: string) {
```

Comment:
We would prefer to use the `validateGeneratedSpec` function instead, so we can make the assertion part more explicit and readable for each test. Besides, in that way we make sure we're following the same assertion style used to test the rest of asyncapi commands so far.